client/src/Categories/CategoriesSources.js

```diff
@@ -0,0 +1,30 @@
+import { Link } from 'react-router-dom';
+import { useParams } from 'react-router-dom';
+
+import './CategoriesList.scss';
+import { useEffect, useState } from 'react';
+import Api from '../Api';
+
+function CategoriesList() {
```

Comment:
You should make the primary function component name match the filename... CategoriesList != CategoriesSources